manifests/charts/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: istiod-cr{{- if not (eq .Values.revision "")}}-{{ .Values.revision }}{{- end }}{{ .Release.Namespace }}
```

Comment:
Can we remove this ? I don't think we need hungarian notation - kind=ClusterRole.